### ISSUE TITLE
Can it be be DECODED? & html support.

### ISSUE THREAD

**Issue by gr8rehanna** [arshidkv12/phpBolt#25]
This tool is amazing..
But it only encrypts PHP and if any line of HTML code is inside page is shown normally.

Most important questions.
After encryption key is place in file.

Can someone get the original source of php??

**Comment by naifamoodi** [comment]
I have the same question. How secure is this? If you had to rate from 1 to 10, where 10 is the highest, what would your rating be as far as its security is concerned?

**Comment by gr8rehanna** [comment]
I dont think he is going to rate or provide any answer.
I tried to decode but is not easy enough.

**Comment by dgbaopro2407** [comment]
I can easily decode it

**Comment by gflaminio3** [comment]
> I can easily decode it

yes but, proofs?

**Comment by gr8rehanna** [comment]
Seems it is not as easy people think.
I encoded a file with bolt and same file with ioncube.

while ioncube file was decoded on the go.
bolt encrypted file is not yet decoded i have sent files to few of my friends they tried enough but no success as of now.

Anyone looking for answers: CAN SOMEONE DECODE BOLT?
Answer: Maybe YES but NO - It is not that easy for now.

**Comment by Dichvucoder** [comment]
> Seems it is not as easy people think. I encoded a file with bolt and same file with ioncube.
> 
> while ioncube file was decoded on the go. bolt encrypted file is not yet decoded i have sent files to few of my friends they tried enough but no success as of now.
> 
> Anyone looking for answers: CAN SOMEONE DECODE BOLT? Answer: Maybe YES but NO - It is not that easy for now.

https://fb.watch/hKJqwrmMNX/
Is there anything harder ?
Contact me : facebook : https://facebook.com/dgbaopro2407

**Comment by Dichvucoder** [comment]
> > I can easily decode it
> 
> yes but, proofs?

https://fb.watch/hKJqwrmMNX/
Is there anything harder ?
Contact me : facebook : https://facebook.com/dgbaopro2407

**Comment by gr8rehanna** [comment]
Well someone else is decoding these files. \
That too on an Android. Its not a proof actually tbh.

If that easy i have published a project encoded with phpBolt .

People ask me to encode with ionCube even one of user provided me ioncube encoder.\
Just because ioncube encoded files are easily decoded.

For now phpBolt is good to go option.\
I have enough expertise to encode and decode things and i ofcourse tried to decode phpBolt it was not easy.

Final verdict: phpBolt can be decoded but not by everyone especially who steals aka copy/paste codes.(mostly Indian)

For now it is safe untill someone publish a workaround to decode.

ps: i was able to decode after about a month of tries and it was not a decode - I was able to achieve this by creating a php function that generates 500 error to any encoded.php files and then log is saved in error.log file with all file contexts.

Now i dont understand how you /your Facebook guy decoded.

:) It doesn't make sense to discuss further here.\
I got want i was looking for.\
Closing this now.
